wls-gui-wahllokalsystem/src/stores/stimmabgabevermerkeStore.ts

```diff
@@ -159,6 +159,33 @@ export const useStimmabgabevermerkeStore = defineStore(
       );
     }
 
+    function getBlattnummernThatPreventDeletion(newRowSize: number) {
+      const numbers: Set<number> = new Set<number>();
+      const currentLowestNumberOfRowsOverAllWahldaten =
+        lowestNumberOfRowsOverAllWahldaten.value;
+      if (newRowSize > 0 && currentLowestNumberOfRowsOverAllWahldaten) {
+        const removeRows =
+          newRowSize - currentLowestNumberOfRowsOverAllWahldaten - 1;
+        stimmabgabevermerke.value.forEach(
+          (stimmabgabevermerk: Stimmabgabevermerke) => {
+            const vermerke = stimmabgabevermerk.wahldaten[0].vermerke;
+            for (let i = removeRows; i < 0; i++) {
+              const vermerk = vermerke[vermerke.length + i];
+              if (
+                vermerk.stimmzettel.some(
+                  (stimmzettel) =>
+                    stimmzettel.anzahl != null && stimmzettel.anzahl != 0
+                )
+              ) {
+                numbers.add(vermerk.blattnummer);
+              }
```

Comment:
ich denke du kannst das if komplett weg lassen:

```suggestion
              numbers.add(vermerke[vermerke.length + i].blattnummer);
```

Reply:
weil du in der komponente schon bevor du `getBlattnummernThatPreventDeletion` aufrufst mit `isAnyRowThatShouldBeDeletedFilled` geprüft hast, ob eines der inputfelder einen wert hat und das ist ja im prinzip die gleiche logik wie in dem if

Reply:
Ich habe die Überprüfung an der Stelle nochmal, damit beim entfernen mehrerer Zeilen trotzdem noch die leeren Zeilen, die kein Problem darstellen, ausgeschlossen werden. 

<img width="1272" height="837" alt="grafik" src="https://github.com/user-attachments/assets/eabc7b76-6d89-496d-99ba-0ff6a48c4fa9" />

In dem Fall sind z.B. nur Nr. 3 und 5 das Problem. Ohne das if würden im Dialog aber 3, 4 und 5 angezeigt werden